tests/test.py

```diff
@@ -59,6 +59,7 @@
 from peekaboo.toolbox.cuckoo import CuckooReport
 from peekaboo.toolbox.ole import Oletools
 from peekaboo.toolbox.file import Filetools, FiletoolsReport
+from peekaboo.toolbox.known import Knowntools, KnowntoolsReport
```

Comment:
Why do these become necessary now and not before? Should this be squashed into the previous commit?